.devcontainer/images/hooks/lifecycle/postCreate.sh

```
@@ -0,0 +1,347 @@
+
+  if [ -n "$gpg_key" ]; then
+    git config --global user.signingkey "$gpg_key"
+    git config --global commit.gpgsign true
+    git config --global tag.forceSignAnnotated true
+    git config --global gpg.program gpg
+    log_success "Git GPG signing configured with key: $gpg_key (matching $git_email)"
+  else
+    # No matching key found - GPG signing will be configured via /git skill
+    log_warning "No GPG key found for email '$git_email' - configure via /git skill"
+  fi
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)